src/utils/integrationTestIds.js

```diff
@@ -7,6 +7,7 @@ const dataTestId = {
   PROJECT_NAME: 'project-name',
   PROJECT_DESCRIPTION: 'project-description',
   LAUNCH_ANALYSIS_BUTTON: 'launch-analysis-button',
+  QC_STATUS_TEXT: 'qc-status-text',
 };
 
 const dataTestClass = {
```

Comment:
Why not just call this `class`? and dataTestId `id`. That way we don't have so many different names for the same thing

Reply:
And also maybe even `classes` and `ids`, since the file is called `integrationTestIds`in plural too.

Reply:
either that or change the file to `integrationTestId`